contrib/src/lib.rs

```diff
@@ -80,3 +80,14 @@ mod uuid;
 
 #[cfg(feature = "uuid")]
 pub use uuid::{UUID, UuidParseError};
+
+#[cfg(feature = "diesel_sqlite")]
+pub extern crate diesel;
+
+#[cfg(feature = "pg_pool")]
```

Comment:
Note that the `cfg` applies only to the item directly below it. As you've written it, the `r2d2_postgres` crate is always depended on.

Reply:
That's a good idea. I was not super familiar with the way `cfg` works in that way. It makes sense that it only applies to the line directly below. Definitely something that needs to be fixed. Your suggestion makes a lot of sense. It also moves a lot of the syntactic noise to another location so as to not distract from the `lib.rs` file for the `contrib` crate.